modules/service-cc.sh

```diff
@@ -0,0 +1,66 @@
+# shellcheck disable=SC2034,SC2039
+ 
+CC_SERVICE_TITLE="Canonical Common Criteria"
+CC_SUPPORTED_SERIES="xenial"
+CC_SUPPORTED_ARCHS="x86_64 ppc64le s390x"
+
+CC_REPO_URL="https://private-ppa.launchpad.net/fips-cc-stig/fipsdevppa"
```

Comment:
This needs to be the final PPA, even if it's not ready yet. For testing we can switch locally.

Reply:
That being said, can I get access to your testing ppa? :)

Reply:
I just added you to the test ppa. Can you please check access

Reply:
Got it, thanks

Reply:
I updated the ppa to UA/common-criteria and also updated that ppa key in keyrings.